vassal-app/src/main/java/VASSAL/launch/Launcher.java

```diff
@@ -162,6 +167,45 @@ else if (t instanceof ExtensionsLoader.LoadExtensionException ||
     }
   }
 
+
+  private static class LoadConstraint {
+    public String moduleName;
+    public String minVassalVersion;
+    public String minModuleVersion;
+    public String newVersionLink;
+
+    LoadConstraint(String moduleName, String minVassalVersion, String minModuleVersion, String newVersionLink) {
+      this.moduleName = moduleName;
+      this.minVassalVersion =  minVassalVersion;
+      this.minModuleVersion =  minModuleVersion;
+      this.newVersionLink   =  newVersionLink;
+    }
+  }
+
+  private static final List<LoadConstraint> constraints = Arrays.asList(
```

Comment:
`List.of()`

Reply:
Actually, if you're looking these up by name, they should really be in a `Map<String,LoadConstraint>` instead.

Reply:
fixed